test/docker-compose.yml

```diff
@@ -362,7 +362,7 @@ services:
       - --log.level=debug
       # already existed service
       - --node.rpc=http://op-seq:9545
-      - --execution.rpc=http://op-${SEQ_TYPE}-seq:8545
+      - --execution.rpc=http://${SEQ_L2_EXECUTION_CLIENT}:${SEQ_L2_EXECUTION_PORT}
```

Comment:
seems unnecessary to add `SEQ_L2_EXECUTION_PORT `, because it is a port inside a docker container

Reply:
ok, got it